tests/basic/pure_test.go

```diff
@@ -4132,3 +4132,111 @@ var _ = Describe("{CreateNewPoolsWhenFadaFbdaVolumeCreationInProgress}", func()
 	})
 
 })
+
+var _ = Describe("{CreateCloneOfTheFADAVolume}", func() {
+	/*
+				https://purestorage.atlassian.net/browse/PTX-24002
+			    1.Deploy a FADA app
+		        2.Take the corresponding pxctl volume of the pvc and try to clone it
+		        3.Check the corresponding volume clone is available in FA backend
+		        4.Delete the FADA app
+
+	*/
+	JustBeforeEach(func() {
+		log.Infof("Starting Torpedo tests ")
+		StartTorpedoTest("CreateCloneOfTheFADAVolume",
+			"Create Clone of the FADA Volume and verify the status",
+			nil, 0)
+	})
+	itLog := "Px Volume Resize in parallel to FADA/FBDA Volume Resize ( PVC Resize )"
+	It(itLog, func() {
+		var contexts []*scheduler.Context
+		var volumeName string
+		var cloneVolumeId string
+		flashArrays, err := GetFADetailsUsed()
+		log.FailOnError(err, "Failed to get FA details used")
+		log.InfoD(itLog)
+		stepLog := "Deploy FADA app"
+		Step(stepLog, func() {
+			log.InfoD(stepLog)
+			context, err := Inst().S.Schedule("deploy-fada", scheduler.ScheduleOptions{
+				AppKeys:            []string{"fio-fa-davol"},
+				StorageProvisioner: fmt.Sprintf("%v", portworx.PortworxCsi),
+				PvcSize:            6 * units.GiB,
+			})
+			log.FailOnError(err, "Failed to schedule application of %v namespace", "deploy-fada-volume")
```

Comment:
Don't see a namespace passed at the time of deploying the app in previous call.  Double check this once